spec/acceptance/azure_setup_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper_acceptance'
+
+
+describe 'azure class' do
+  azurepackages = ['retries', 'azure', 'azure_mgmt_compute', 'azure_mgmt_storage','azure_mgmt_resources', 'azure_mgmt_network', 'hocon']
+
+#  case fact('osfamily')
```

Comment:
Cool. You pass in the packages based on osfamily in the pararms. This isn't needed is it?